Cmdr/CmdrClient/init.lua

```diff
@@ -49,12 +49,18 @@ function Cmdr:SetActivationKeys (keysArray)
 	self.ActivationKeys = Util.MakeDictionary(keysArray)
 end
 
---- Sets the place name label on the interface
-function Cmdr:SetPlaceName (name)
-	self.PlaceName = name
+--- Sets the prompt on the interface
+function Cmdr:SetPrompt (prompt)
+	self.Prompt = prompt
 	Interface.Window:UpdateLabel()
 end
 
+--- Sets the place name label on the interface
```

Comment:
Replace with 

```lua
Cmdr.SetPlaceName = Cmdr.SetPrompt
```

Reply:
Fixed in https://github.com/evaera/Cmdr/pull/98/commits/18f76d59c5a9d4c79b14c1845a9c2591b4313f8f